mungegithub/mungers/rebuild.go

```diff
@@ -19,12 +19,13 @@ package mungers
 import (
 	"fmt"
 	"regexp"
+	"strings"
 
 	"k8s.io/contrib/mungegithub/features"
 	"k8s.io/contrib/mungegithub/github"
 
 	"github.com/golang/glog"
-	githubapi "github.com/google/go-github/github"
```

Comment:
FWIW the runonname is more go-style than nicely_formatted_name, but I don't care